packages/valory/skills/twitter_scoring_abci/behaviours.py

```diff
@@ -390,9 +397,14 @@ def update_ceramic_db(self, api_data: Dict) -> Dict:
         # entries on the database
         ceramic_db.merge_by_wallet()
 
-        # latest_mention_tweet_id
+        # latest_mention_tweet_id and latest_hashtag_tweet_id
+        latest_id = max(
+            int(api_data["latest_mention_tweet_id"]),
+            int(api_data["latest_hashtag_tweet_id"]),
+        )
+
         ceramic_db.data["module_data"]["twitter"]["latest_mention_tweet_id"] = str(
-            api_data["latest_mention_tweet_id"]
+            latest_id
```

Comment:
This fixes the issue: we need to check BOTH tweet ids, see which one is the max and save it.